src/openms/source/ANALYSIS/TARGETED/TargetedExperiment.cpp

```diff
@@ -98,6 +98,26 @@ namespace OpenMS
   {
   }
 
+  TargetedExperiment::TargetedExperiment(TargetedExperiment && rhs) noexcept :
+    cvs_(std::move(rhs.cvs_)),
+    contacts_(std::move(rhs.contacts_)),
+    publications_(std::move(rhs.publications_)),
+    instruments_(std::move(rhs.instruments_)),
+    targets_(std::move(rhs.targets_)),
+    software_(std::move(rhs.software_)),
+    proteins_(std::move(rhs.proteins_)),
+    compounds_(std::move(rhs.compounds_)),
+    peptides_(std::move(rhs.peptides_)),
+    transitions_(std::move(rhs.transitions_)),
+    include_targets_(std::move(rhs.include_targets_)),
+    exclude_targets_(std::move(rhs.exclude_targets_)),
+    source_files_(std::move(rhs.source_files_)),
+    protein_reference_map_dirty_(true),
+    peptide_reference_map_dirty_(true),
+    compound_reference_map_dirty_(true)
```

Comment:
@timosachsenberg the maps are empty and marked as "dirty" so they will not be used but rather re-built